app/grandchallenge/reader_studies/views.py

```diff
@@ -959,6 +971,22 @@ def _check_change_perms(self, user, obj):
         if not (user and user.has_perm(self.change_permission, obj)):
             raise Http404()
 
+    def get_permissions(self, *args, **kwargs):
+        if self.action == "remove_ground_truth":
+            return [DeleteGroundTruthPermission()]
+        return super().get_permissions(*args, **kwargs)
+
+    @action(detail=True, url_path="delete-ground-truth", methods=["post"])
```

Comment:
POST for deletion? Why use DRF for this? Wouldn't a normal view be better, then it could be hooked up with HTMX if needed.

Reply:
I went through the options for what type of request to use and came across this: https://itnext.io/mass-delete-via-http-rest-how-do-you-do-it-1bff0f5eb72d. The arguments there made sense to me, which is why I opted for POST rather than DELETE. I think moving this to a regular view makes sense though.

Reply:
Interesting, but that is a bit out of date, it looks like OpenAPI now supports request bodies with DELETE https://github.com/OAI/OpenAPI-Specification/pull/1937 so I would say we could/should use it in this situation.